File: product-app-spring-2/src/main/java/com/training/pms/services/ProductService.java
package com.training.pms.services;

import java.util.List;

import com.training.pms.model.Product;

public interface ProductService {
	public boolean addProduct(Product product);
	public boolean updateProduct(int productId, Product product);
	public boolean deleteProduct(int productId);
	public Product findByProduct(int productId);
	public List<Product> findByProduct(String productName);
	public List<Product> findByProduct();
}

File: product-app-rakuten/src/services/ProductService.js
import axios from 'axios'

const PRODUCT_API_URL = 'http://localhost:9090/product'
class ProductService {

    retrieveAllProducts() {
        return axios.get(`${PRODUCT_API_URL}`);
    }

    deleteProduct(productId) {
        return axios.delete(`${PRODUCT_API_URL}/${productId}`);
    }
}

export default new ProductService()